contributor/01-introduction-article.md

```diff
@@ -0,0 +1,97 @@
+# Draft comments
+
+ Please watch the [Intro to InnerSource Video](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504)
+* Also watch the [Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925) to the Trusted Committer role.
+* The [Lowering the Barriers to Entry](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323929) video of the Trusted Committer role can be a good addition to understand more of why the Trusted Committer might ask for different things.
+
+## Outline based Target
+### The Contributor in InnerSource
+
+  - Introduce the speaker. - only required for the video script I'd say. 
+  - Quick overview of the _Contributor_ role.
+  - Summary of the roles of InnerSource and how each relates to _Contributor_.
+  - Summary of the upcoming segments and what each covers.
+
+Outline for this section inspired by the [Trusted Committer introduction article.
+
+## Video Play-book based Target
+How do I best contribute to an InnerSource project?  This segment shares a summary of what it means to be an InnerSource Contributor.  We'll also give an overview of the upcoming training segments on being a great InnerSource Contributor.
+
+
+
+# Article
+
+## The Contributor in InnerSource.
+
+Without contributions, an InnerSource community and its project(s) would not exist in a meaningful way.
+There would be open repositories, ideally equipped with somewhat specific extra documents (e.g. a CONTRIBUTING.md and a README.md) and person(s) committing to them, e.g. a Trusted Committer.
+[EDITOR NOTE:
+- In other articles we draw a distinction between guest and host team.
+- We argue that the Committer is part of the guest team and needs to fit in with the host team.
+- In a more-than-one-person project what do we call the people aside of the TC/PO that are members of the host team? Just team members?
+        - OSS would likely not make this distinction and everyone would end up as contributor until it's committer / trusted committer at some point I guess?
+        - In the ASF-Rule-set this would be the distinction between Developer and Committer.
```

Comment:
This distinction is taken from https://www.apache.org/foundation/how-it-works.html#developers